### PR TITLE
Add Quora

### PR DESCRIPTION
**Pull request by SandeepTuniki**
**[https://github.com/SandeepTuniki/awesome-quora/]**

**[`awesome-quora` is a vast collection of interesting questions, answers and blog posts on Quora. It contains quality answers written by people across the world, many of whom are experts in their respective fields. The project's aim is to provide an interesting read for people looking for simple explanations of how/why the events in our world happens.]**
# ALL THE BELOW CHECKBOXES SHOULD BE DONE AND CHECKED AND THE ABOVE PLACEHOLDERS SHOULD BE FILLED IN BEFORE SUBMITTING A PULL REQUEST!
- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. _(For example: `Add Node.js`)_
- The list I added
  - [x] **has been around for at least 20 days,**
  - [x] is a non-generated Markdown file in a GitHub repo,
  - [x] is not a duplicate,
  - [x] only has awesome items with descriptions _(Awesome lists are curations of the best, not of everything)_,
  - [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
  - [x] has a Table of Contents section named `Contents` as the first section,
  - [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) _(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)_,
  - [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
  - [x] has been checked for inconsistent formatting, spelling, and grammar _(This includes starting each link description with an uppercase character and ending it with a period `.`, and consistent naming)_,
  - [x] has been added at the bottom of the appropriate category.

### CODE REVIEW

**Comment by davisonio** [comment]
> is not a duplicate,

We already have https://github.com/cyberglot/awesome-answers which also includes Quora answers.


> only has awesome items with descriptions (Awesome lists are curations of the best, not of everything),

There's a lot of items on this list which has clearly taken lots of effort to add here :smile: but they don't seem to be there in any structured way. Ideally there should be categories for the questions and a description why the answer is awesome and interesting.

I would also remove the separators - they don't look so great.

**Comment by SandeepTuniki** [comment]
@davisonio 

**Regarding the duplicate:** I am aware of the `awesome-answers` respository that you mentioned. But it solely focuses on programming related questions. My repo, on the other hand, tries to gather links from all disciplines. In fact, very few of the links in my repo are related to programming. So, there is less overlap between the two projects than the titles suggest. :simple_smile:

**Regarding the Structuring:** This was deliberately done. A lot of questions are general questions which are very hard to categorize (e.g: _What are the key traits of mentally strong people?_). They would all go into the **Miscellaneous** category if I try to categorize them. Still, to enforce some order among them, I have sorted all the questions in alphabetical order.

**Regarding the separators:** Removed them.

**Comment by SandeepTuniki** [comment]
ping @davisonio 
Any update to my pull request?

**Comment by davisonio** [comment]
I think that a few non-question resources would be great, to make this a complete Quora list. e.g. links to their mobile apps, social media, subreddit.

File: readme.md [hunk 1]
```diff
@@ -445,6 +445,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [DTrace](https://github.com/xen0l/awesome-dtrace) - Dynamic tracing framework.
 - [Userscripts](https://github.com/brunocvcunha/awesome-userscripts) - Enhance your browsing experience.
 - [Pokémon](https://github.com/tobiasbueschel/awesome-pokemon) - Pokémon and Pokémon GO resources and tools.
+- [Quora Collection](https://github.com/SandeepTuniki/awesome-quora) - Collection of interesting questions, answers & blog posts on Quora
```
Comment (davisonio) [inline readme.md]: I think I would change this to:
`- [Quora](https://github.com/SandeepTuniki/awesome-quora) - Interesting questions, answers & blog posts on Quora`

Comment (sindresorhus) [inline readme.md]:
```markdown
- [Quora](https://github.com/SandeepTuniki/awesome-quora) - Interesting questions, answers, and blog posts on Quora.
```

Comment (davisonio) [inline readme.md]: @SandeepTuniki

Comment (SandeepTuniki) [inline readme.md]: @davisonio 
Sorry for not responding quickly. I've had a heck of a lot of work in the last month, and didn't get time to work on the pull request. I will implement your suggestions this weekend.

**Comment by sindresorhus** [comment]
ping

**Pull request closed without merging**